packages/rss-handler/src/index.ts

```diff
@@ -121,15 +149,12 @@ const parser = new Parser({
   timeout: 60000, // 60 seconds
   maxRedirects: 10,
   customFields: {
-    item: [['link', 'links', { keepArray: true }], 'published', 'updated'],
-    feed: ['dc:date', 'lastBuildDate', 'pubDate'],
-  },
-  headers: {
-    // some rss feeds require user agent
-    'User-Agent':
-      'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/116.0.0.0 Safari/537.36',
-    Accept:
-      'application/rss+xml, application/rdf+xml;q=0.8, application/atom+xml;q=0.6, application/xml;q=0.4, text/xml;q=0.4',
+    item: [
+      ['link', 'links', { keepArray: true }],
+      'published',
+      'updated',
+      'created',
+    ],
```

Comment:
the custom fields for feed are missing
`feed: ['dc:date', 'lastBuildDate', 'pubDate']`

Reply:
weird i wonder why this diff got all messed up, thanks for catching